src/Message/Formatter.php

```diff
@@ -141,7 +141,19 @@ private function defaultFormat(Message $message, array $commonContext): string
     private function getTime(Message $message): string
     {
         $timestamp = (string) $message->context('time', microtime(true));
-        $format = strpos($timestamp, '.') === false ? 'U' : 'U.u';
+
+        switch (true) {
+            case strpos($timestamp, '.') !== false:
+                $format = 'U.u';
+                break;
+            case strpos($timestamp, ',') !== false:
+                $format = 'U,u';
+                break;
+            default:
+                $format = 'U';
```

Comment:
`U` is a common part, can be extracted to something like `$prefix` or `$baseMessage` and then concatenated with unique part.

Reply:
done.

Reply:
Can't see any changes related to this.

Reply:
Decimal separator can be obtained with localeconv()['decimal_point'].